Scala 2.13.18 release notes (DRAFT) (#920)
scala/scala-dev: 2.13.18.md

# Scala 2.13.18 (DRAFT)

The Scala team at Akka is pleased to announce Scala 2.13.18.

> This release fixes a number of regressions from 2.13.17. It is compatible with JDKs 8 to 26.

Reviewer: ```suggestion
This release fixes several regressions from 2.13.17. It is compatible with JDKs 8 to 26.
```
